packages/react-component-library/src/components/DatePicker/FloatingCalendar.tsx

```diff
@@ -0,0 +1,272 @@
+import {
+  addMonths,
+  setMonth as changeMonth,
+  setYear as changeYear,
+} from 'date-fns/fp'
+import { enGB } from 'date-fns/locale'
+import { format } from 'date-fns'
+import FocusTrap from 'focus-trap-react'
+import React, { memo, useCallback } from 'react'
+
+import { BUTTON_VARIANT } from '../Button'
+import {
+  CALENDAR_TABLE_VARIANT,
+  CalendarTableVariant,
+  DATE_VALIDITY,
+} from './constants'
+import { CalendarNavigation } from './CalendarNavigation'
+import { CalendarTable } from './CalendarTable'
+import { COMPONENT_SIZE } from '../Forms'
+import { DATEPICKER_ACTION } from './types'
+import { DatePickerProps } from './DatePicker'
+import { replaceInvalidDate } from './utils/isDateValid'
```

Comment:
Import can be shortened here:
```suggestion
import { replaceInvalidDate } from './utils'
```